devify

```diff
@@ -48,8 +48,8 @@ case "$1" in
 *"Mouse"* | *"Logitech MX"*)
   notify "$1" "generic_mouse"
   ;;
-*"X-Box"*)
-  notify "X-Box Controller" "xbox_controller"
+*"X-Box"* | *"Xbox"*)
+  notify "Xbox Controller" "xbox_controller"
```

Comment:
Microsoft uses the branding "Xbox", not "X-Box".

Reply:
> For future reference, my controllers showed up as `Xbox Wireless Controller` and `Pro Controller`.

Interesting. Mine shows up as "X-box" (with a dash). Hence why I named it liked that. 
You're right though, it does seems more appropriate